README.md

```diff
@@ -230,6 +230,16 @@ To create a new instance of SampleObject
 SampleObject.build({id: 'some id', field_one: 'Some value'})
 ```
 
+## Troubleshooting
+
+If you get following error on a threaded server
+
+```
+ActiveRecord::ConnectionTimeoutError: could not obtain a database connection within 5 seconds (waited 5.000000001 seconds). The max pool size is currently 5; consider increasing it.
+```
+
+Consider setting environment variable `NULLDB_MAX_POOL_SIZE` to a value greater than the max thread pool size.
```

Comment:
This looks like a temporary fix. We wanna find a better way to establish a **dummy** connection to `NULLDB`. Either by forking NULLDB or using some config.